slither/core/cfg/node.py

```diff
@@ -230,6 +230,10 @@ def __init__(
         self.scope: Union["Scope", "Function"] = scope
         self.file_scope: "FileScope" = file_scope
 
+        # Record the continue destination in the STARTLOOP node, so `continue` node can
+        # find the correct jump destination.
+        self.continue_destination: Node = None
```

Comment:
Would it be able to pass down the break and continue context in the parsing rather than store the data on the node?

Reply:
Yes, once https://github.com/crytic/slither/pull/629 is merged, I can use `_loops`.
Maybe I can create a new PR on top of `feature/cleanup-fixup` branch.